omnigibson/robots/stretch.py

```diff
@@ -0,0 +1,203 @@
+                GraspingPoint(link_name="r_gripper_finger_link", position=[-0.025, -0.012, 0.0]),
+            ]
+        }
+
+    @property
+    def assisted_grasp_end_points(self):
+        return {
+            self.default_arm: [
+                GraspingPoint(link_name="l_gripper_finger_link", position=[0.025, 0.012, 0.0]),
+                GraspingPoint(link_name="l_gripper_finger_link", position=[-0.025, 0.012, 0.0]),
+            ]
+        }
+
+    @property
+    def disabled_collision_pairs(self):
+        return [
+            ["base_link", "caster_link"],
+            ["base_link", "link_aruco_left_base"],
+            ["base_link", "link_aruco_right_base"],
+            ["base_link", "base_imu"],
+            ["base_link", "laser"],
+            ["base_link", "link_left_wheel"],
+            ["base_link", "link_right_wheel"],
+            ["base_link", "link_mast"],
+            ["link_mast", "link_head"],
+            ["link_head", "link_head_pan"],
+            ["link_head_pan", "link_head_tilt"],
+            ["camera_link", "link_head_tilt"],
+            ["camera_link", "link_head_pan"],
+            ["link_head_nav_cam", "link_head_tilt"],
+            ["link_head_nav_cam", "link_head_pan"],
+            ["link_mast", "link_lift"],
+            ["link_lift", "link_aruco_shoulder"],
+            ["link_lift", "link_arm_l4"],
+            ["link_lift", "link_arm_l3"],
+            ["link_lift", "link_arm_l2"],
+            ["link_lift", "link_arm_l1"],
+            ["link_arm_l4", "link_arm_l3"],
+            ["link_arm_l4", "link_arm_l2"],
+            ["link_arm_l4", "link_arm_l1"],
+            ["link_arm_l3", "link_arm_l2"],
+            ["link_arm_l3", "link_arm_l1"],
+            ["link_arm_l2", "link_arm_l1"],
+            ["link_arm_l0", "link_arm_l1"],
+            ["link_arm_l0", "link_arm_l2"],
+            ["link_arm_l0", "link_arm_l3"],
+            ["link_arm_l0", "link_arm_l4"],
+            ["link_arm_l0", "link_arm_l1"],
+            ["link_arm_l0", "link_aruco_inner_wrist"],
+            ["link_arm_l0", "link_aruco_top_wrist"],
+            ["link_arm_l0", "link_wrist_yaw"],
+            ["link_arm_l0", "link_wrist_yaw_bottom"],
+            ["link_arm_l0", "link_wrist_pitch"],
+            ["link_wrist_yaw_bottom", "link_wrist_pitch"],
+            ["gripper_camera_link", "link_gripper_s3_body"],
+            ["link_gripper_s3_body", "link_aruco_d405"],
+            ["link_gripper_s3_body", "link_gripper_finger_left"],
+            ["link_gripper_finger_left", "link_aruco_fingertip_left"],
+            ["link_gripper_finger_left", "link_gripper_fingertip_left"],
+            ["link_gripper_s3_body", "link_gripper_finger_right"],
+            ["link_gripper_finger_right", "link_aruco_fingertip_right"],
+            ["link_gripper_finger_right", "link_gripper_fingertip_right"],
+            ["respeaker_base", "link_head"],
+            ["respeaker_base", "link_mast"],
+        ]
```

Comment:
Do we actually need to disable this many collisions?

Reply:
Yeah I manually went through all of them...ideally we should just delete the links themselves, since a lot of the links are very poorly modeled and very trivial, but I was too lazy and wanted to keep it consistent with the raw URDF file LOL